src/app/Services/MessageService.ts

```diff
@@ -40,6 +40,7 @@ export class MessageService {
 
     public OnMessage(data: MessageEvent) {
         const ev = JSON.parse(data.data) as AiurEvent;
+        const fireAlert  = window.hasOwnProperty('cordova') || this._electronService.isElectronApp;
```

Comment:
I am considering. What if the user is using it from PWA or web but there was no google available for him? Like a typical Chinese user?

I guess the `fireAlert` shall only set to false if the current device is successfully subscribed to the push service. Otherwise, always true.